x/wasm/keeper/keeper_test.go

```diff
@@ -530,6 +545,12 @@ func TestExecute(t *testing.T) {
 	require.NotNil(t, contractAcct)
 	assert.Equal(t, sdk.Coins(nil), bankKeeper.GetAllBalances(ctx, contractAcct.GetAddress()))
 
+	// and events emitted
+	require.Len(t, em.Events(), 5)
+	expEvt := sdk.NewEvent("execute",
```

Comment:
event 0 is the message and the other ones?
I find it nice to test all the types at least, makes it clear what is (expected to) happen